tests/builtin_server/tests.py

```diff
@@ -81,15 +81,15 @@ class WSGIFileWrapperTests(TestCase):
 
     def test_file_wrapper_uses_sendfile(self):
         env = {"SERVER_PROTOCOL": "HTTP/1.0"}
-        handler = FileWrapperHandler(None, BytesIO(), BytesIO(), env)
+        handler = FileWrapperHandler(BytesIO(), BytesIO(), BytesIO(), env)
```

Comment:
These should never have been `None`. The `stdin` value is passed to become `wsgi.input` which should always be a file-like object.